cogip/widgets/gameview.py

```diff
@@ -415,17 +415,45 @@ def plane_released(self, pick: Qt3DRender.QPickEvent):
         self.plane_intersection = None
         self.new_move_delta.emit(None)
 
-    def new_robot_state(self, new_state: models.RobotState) -> None:
+    def add_robot(self, robot_id: int) -> None:
+        """
+        Add a new robot status bar.
+
+        Parameters:
+            robot_id: ID of the new robot
+        """
+        if self.path.get(robot_id):
+            return
+        path = PathEntity(parent=self.scene_entity)
+        path.set_points([])
+        self.path[robot_id] = path
+
+    def del_robot(self, robot_id: int) -> None:
+        """
+        Remove a robot.
+
+        Parameters:
+            robot_id: ID of the robot to remove
+        """
+        path = self.path.pop(robot_id, None)
+        if not path:
+            return
+        path.set_points([])
+
+    def new_robot_state(self, robot_id: int, new_state: models.RobotState) -> None:
         """
         Function called when robot state is updated.
         Update computed path.
 
         Arguments:
             new_state: new robot state
         """
+        path = self.path.get(robot_id)
```

Comment:
missing `robot_id` description in method doc.

Reply:
Done